### PR TITLE
Trim mime db

### LINKED ISSUE #2099: Trim down mime-db [1]

### PR DESCRIPTION
**Pull request by molant**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- ~~Added/Updated related documentation.~~
- ~~Added/Updated related tests.~~

## Short description of the change(s)

Fix #2099

### CODE REVIEW

**Comment by molant** [comment]
@antross the failures in `http-compression` were caused because you unpinned the version of mime-db. Tests should be passing now and hopefully ready to merge!

**Comment by antross** [comment]
> @antross the failures in http-compression were caused because you unpinned the version of mime-db. Tests should be passing now and hopefully ready to merge!

🤦‍♂️ Thanks for figuring this out!

File: packages/utils/README.md [hunk 1]
```diff
@@ -41,13 +41,28 @@ and exception.
 ### dom
 
 * `createHTMLDocument`: Create an HTMLDocument object from an string.
+* `determineMediaTypeBasedOnFileExtension`: Get a mime-type associated
```
Comment (antross) [inline packages/utils/README.md]: Shouldn't these be under a `### contentType` section?

Comment (molant) [reply packages/utils/README.md]: Yeah... bad rebase. Nice catch!

**Comment by molant** [comment]
This should be ready for another review

**Comment by antross** [comment]
@sarvaje Can you take a look at this when you have time? Thanks!

File: packages/utils/src/content-type.ts [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
-import { normalizeString } from '@hint/utils/dist/src/misc/normalize-string';
-import { HTMLElement } from '@hint/utils/dist/src/dom/html';
+import { HttpHeaders } from './types/http-header';
+import { debug as d } from './debug';
+import { fileExtension as getFileExtension, fileName as getFileName } from './fs';
+import { normalizeString } from './misc/normalize-string';
 
 const debug = d(__filename);
 
+type HTMLElement = {
```
Comment (sarvaje) [inline packages/utils/src/content-type.ts]: This shouldn't be necessary, HTMLElement is now in the @hint/utils package.

Comment (molant) [reply packages/utils/src/content-type.ts]: This should be fixed now!

**Pull request merged**